<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>process_showsClass</class>
 <widget class="QMainWindow" name="process_showsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>媒体刮削器</string>
  </property>
  <widget class="QWidget" name="Widget1">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="libraryText">
          <property name="text">
           <string>搜索到的路径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="libraryTree">
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>路径</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="ignoreText">
          <property name="text">
           <string>被忽略的路径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="ignoreList">
          <property name="selectionMode">
           <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Expand">
        <property name="text">
         <string>展开</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Collapse">
        <property name="text">
         <string>收缩</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Waiting">
        <property name="text">
         <string>请等待搜索完成</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="CreateIgnore">
        <property name="toolTip">
         <string>为选中的被忽略的路径创建 .ignore 文件</string>
        </property>
        <property name="text">
         <string>创建 .ignore 文件</string>
        </property>
		<property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Next">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>下一步</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
